.github/workflows/ci.yaml

```diff
@@ -14,9 +14,6 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v4
-      - uses: bufbuild/buf-setup-action@v1.29.0
```

Comment:
Is this the only place we use it?

Reply:
Ugh, no. `ag` doesn't search hidden folders (`.github`) by default and my search missed it in `cloudflare.yml`. Fixed.